Jasmine/Common/Models/CharacterGrid.swift

```diff
@@ -0,0 +1,83 @@
+struct CharacterGrid {
```

Comment:
`CharacterGrid` may not be a good representation of what it is capable of storing (short text).
`TextGrid`? Or why not just `Grid`?